sql/core/src/test/scala/org/apache/spark/sql/errors/QueryCompilationErrorsSuite.scala

```diff
@@ -17,17 +17,17 @@
 
 package org.apache.spark.sql.errors
 
-import org.apache.spark.{SPARK_DOC_ROOT, SparkUnsupportedOperationException}
-import org.apache.spark.sql.{AnalysisException, ClassData, IntegratedUDFTestUtils, QueryTest, Row}
+import org.apache.spark.{SPARK_DOC_ROOT, SparkIllegalArgumentException, SparkUnsupportedOperationException}
+import org.apache.spark.sql._
```

Comment:
Could you revert this, please. See https://github.com/databricks/scala-style-guide?tab=readme-ov-file#imports

Reply:
but it is ok if there are ±6 imports.

Reply:
Ya sorry I need to set up my IDE appropriately so it doesn't do this